flask_selfdoc/autodoc.py

```diff
@@ -22,6 +42,87 @@
     get_function_code = attrgetter('__code__')
 
 
+def custom_jsonify(*args,
+                   indent: Optional[int] = None,
+                   separators: Optional[Tuple] = (',', ':'),
+                   **kwargs):
+    response = jsonify(*args, **kwargs)
+    json_data = json.loads(response.data.decode('utf-8'))
+    json_string = json.dumps(json_data,
+                             indent=indent,
+                             separators=separators)
+    response.data = json_string.encode('utf-8')
+    return response
+
+
+def get_decorator_frame_info(frame) -> dict:
```

Comment:
this is the fix for the line numbers that should work with any python version >= 3.6, with some documentation